server/utils/searchYoutube.ts

```diff
@@ -0,0 +1,20 @@
+import Youtube from 'youtube-api'
```

Comment:
if you're moving YT code to its own file, probably also want to move the YT authenticate call here as well